packages/dashboard/src/pages/CreateMaskWallet/components/CreateMnemonic/index.tsx

```diff
@@ -110,15 +110,7 @@ const CreateMnemonic = memo(() => {
         )
 
         await PluginServices.Wallet.updateMaskAccount({ account: address_ })
-
-        const account = await Services.Settings.getSelectedWalletAddress()
```

Comment:
cc @guanbinrui

This `Services.Settings.getSelectedWalletAddress` points to a fake implementation (used for MV3 migration) that always returns `""` (looks like the original implementation has already be removed).

I treat this line as a constant `""`, then the code becomes:

```js
await PluginServices.Wallet.updateMaskAccount({ account: address_ })

if (!"") {
    await PluginServices.Wallet.updateMaskAccount({ account: address_ })
    await PluginServices.Wallet.resolveMaskAccount([address_])
}
```

```js
await PluginServices.Wallet.updateMaskAccount({ account: address_ })

if (true) {
    await PluginServices.Wallet.updateMaskAccount({ account: address_ })
    await PluginServices.Wallet.resolveMaskAccount([address_])
}
```

```js
await PluginServices.Wallet.updateMaskAccount({ account: address_ })
await PluginServices.Wallet.updateMaskAccount({ account: address_ })
await PluginServices.Wallet.resolveMaskAccount([address_])
```

Duplicated call removed.

```js
await PluginServices.Wallet.updateMaskAccount({ account: address_ })
await PluginServices.Wallet.resolveMaskAccount([address_])
```